### PR TITLE
fix dead links

### PR DESCRIPTION
**Pull request by zramsay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,17 +32,17 @@ Please read, understand and adhere to our [code of conduct](CODE_OF_CONDUCT.md).
...
 
-To download pre-built binaries, see our [downloads page](https://tendermint.com/intro/getting-started/download).
+To download pre-built binaries, see our [downloads page](https://tendermint.com/download).
 
 To install from source, you should be able to:
 
 `go get -u github.com/tendermint/tendermint/cmd/tendermint`
 
-For more details (or if it fails), see the [install guide](https://tendermint.com/docs/guides/install).
+For more details (or if it fails), see the [install guide](https://tendermint.com/docs/guides/install-from-source).
```
Comment (zramsay) [inline README.md]: this should be sorted in #575

File: README.md [hunk 2]
```diff
@@ -32,17 +32,17 @@ Please read, understand and adhere to our [code of conduct](CODE_OF_CONDUCT.md).
 
 ## Install
 
-To download pre-built binaries, see our [downloads page](https://tendermint.com/intro/getting-started/download).
+To download pre-built binaries, see our [downloads page](https://tendermint.com/download).
```
Comment (odeke-em) [inline README.md]: that redirects to /downloads so perhaps, let's replace
https://tendermint.com/downloads with https://tendermint.com/downloads?

**Comment by melekes** [comment]
@zramsay you can use `[ci skip]` in your commit message if you don't want to run the test suite

**Comment by melekes** [comment]
Merged in 742b5b705fbccef2a83329d5912db153a8037001. @zramsay thanks!

**Pull request closed without merging**